SPECS/kata-containers-cc/kata-containers-cc.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
   "Signatures": {
-    "kata-containers-cc-3.2.0.azl3-cargo.tar.gz": "ce8f08246d992c514cb5e63f46dcbbdb15d2aa9ac3ddcd2c6a1c5c08151cca34",
-    "kata-containers-cc-3.2.0.azl3.tar.gz": "c606df786c7bf3aa1738ba8abf721fbe4fd4908f0763a6cae0cc1e5cd747569e"
+    "kata-containers-3.2.0.azl3-cargo.tar.gz": "38dd5d2cf22edb75169809978d6578a23d5177590a25dbd97071ff06378ea77d",
```

Comment:
built from github release tarball by following https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/2568/Building-RPMs-on-a-node-and-creating-vendor-tarballs?anchor=vendor-from-public-github-release-sources